src/main/java/app/chatbot/Duke.java

```diff
@@ -0,0 +1,92 @@
+package app.chatbot;
```

Comment:
Hi. I am Arkar and I will be reviewing the coding standard.

Reply:
Thanks for the in-depth review! The changes you suggested are really sound - I'll be tackling them in future commits for this week.